django/test/runner.py

```diff
@@ -463,6 +463,9 @@ def _init_worker(
             process_setup(*process_setup_args)
         django.setup()
         setup_test_environment(debug=debug_mode)
+        call_command(
+            "check", stdout=io.StringIO(), stderr=io.StringIO(), databases=used_aliases
+        )
```

Comment:
I find it a little awkward that we're going through calling the management command to form its output and discard, but after drilling down, I found that we need this to reuse the logic from `BaseCommand.check()` to raise `SystemCheckError` for any "serious" issues. So it's not easy to avoid this step.